src/components/Button/Button.scss

```diff
@@ -54,13 +54,56 @@ $spinner-size: rem(20px);
   margin-left: -($spinner-size / 2);
 }
 
+.pressed {
+  background: linear-gradient(
+    to bottom,
+    color('sky', 'light'),
+    color('sky', 'light')
+  );
+  border-color: color('sky', 'dark');
+  box-shadow: inset 0 1px 1px 0 rgba(color('ink', 'lighter'), 0.1),
+    inset 0 1px 4px 0 rgba(color('ink', 'lighter'), 0.2);
+
+  &:focus {
+    border-color: color('indigo');
+    box-shadow: inset 0 1px 1px 0 rgba(color('ink', 'lighter'), 0.1),
+      inset 0 1px 4px 0 rgba(color('ink', 'lighter'), 0.2),
+      0 0 0 1px color('indigo');
+  }
+
+  @media (-ms-high-contrast: active) {
+    &.pressed {
+      color: ms-high-contrast-color('button-text');
+      background: ms-high-contrast-color('button-text-background');
+    }
```

Comment:
@dpersing The other 3 similar components: radio, checkbox, and select use `-ms-high-contrast` so I added `button-text` and `button-text-background`.

Reply:
@solonaarmstrong That looks awesome! Thank you.